tests/storage/test_state.py

```diff
@@ -1050,14 +1050,14 @@ def test_return_expanded(self):
             ),
         )
 
-        # Member-only state filters stay the same
+        # Wildcard member-only state filters stay the same
         self.assertEqual(
             StateFilter.freeze(
-                {EventTypes.Member: {"@wombat:test", "@alicia:test"}},
+                {EventTypes.Member: None},
                 include_others=False,
             ).return_expanded(),
             StateFilter.freeze(
-                {EventTypes.Member: {"@wombat:test", "@alicia:test"}},
+                {EventTypes.Member: None},
```

Comment:
Thanks---though I think both of these test cases are worth having?

Reply:
The test case just before was a dupe unless I misread.